pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -201,6 +201,12 @@ func (n *nodeHealthMap) set(name string, data *nodeHealthData) {
 	n.nodeHealths[name] = data
 }
 
+func (n *nodeHealthMap) delete(name string) {
```

Comment:
If it is deleted, it needs to be judged not to be nil for the getDeepCopy 

https://github.com/kubernetes/kubernetes/blob/dff5940ac349bf85e0ea365c6f84f9b1d559378c/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L190-L196

Reply:
OK.